frontend/src/app/logs/logs.component.html

```diff
@@ -21,46 +23,57 @@ <h1 class="oib-title">{{ 'logs.title' | translate }}</h1>
             <oib-datetimepicker formControlName="end" id="end"></oib-datetimepicker>
             <val-errors [control]="searchForm"></val-errors>
           </div>
-          <label for="levels" class="col-auto col-form-label" translate="logs.search.levels"></label>
-          <div class="col-1">
-            <oib-multi-select formControlName="levels" id="levels">
-              @for (level of levels; track level) {
-                <oib-multi-select-option [value]="level" [label]="level | logLevelsEnum"></oib-multi-select-option>
-              }
-            </oib-multi-select>
-          </div>
-
-          <label for="scope-types" class="col-auto col-form-label" translate="logs.search.scope-type"></label>
-          <div class="col-2">
-            <oib-multi-select formControlName="scopeTypes" id="scope-types">
-              @for (scopeType of scopeTypes; track scopeType) {
-                <oib-multi-select-option [value]="scopeType" [label]="scopeType | scopeTypesEnum"></oib-multi-select-option>
-              }
-            </oib-multi-select>
-          </div>
+          @if (this.scopeId == null && this.scopeType == null) {
+            <label for="levels" class="col-auto col-form-label" translate="logs.search.levels"></label>
+            <div class="col-1">
+              <oib-multi-select formControlName="levels" id="levels">
+                @for (level of levels; track level) {
+                  <oib-multi-select-option [value]="level" [label]="level | logLevelsEnum"></oib-multi-select-option>
+                }
+              </oib-multi-select>
+            </div>
```

Comment:
Be careful, you hide the "levels" input. Only scope types should be hidden here.

Reply:
Yes, this is on purpose, I create another ‘’level’’ input on the second line if scopeType and scopeId aren’t null. It makes the form prettier and more readable, according to me.